Week4/Blockbuster/app/src/main/java/com/example/blockbuster/data/Movie.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.blockbuster.data
+
+import java.io.Serializable
+
+class Movie(
+    val id: Int, val releaseDate: String,
+    var title: String,
+    val summary: String?,
+    var poster: Any?, var genre: String, var stars: Float) : Serializable
```

Comment:
I'm not entirely sure on this one but I question that perhaps Parcelable or Parcelize should be used instead of Serializable?